pipelines/vars/forklift_katello.yml

```diff
@@ -13,6 +12,7 @@ server_box:
         - "--puppet-server-jvm-min-heap-size 1G"
         - "--puppet-server-jvm-max-heap-size 1G"
         - "{{ '--foreman-proxy-content-enable-ostree true' if (pipeline_os != 'centos7' and (pipeline_version == 'nightly' or pipeline_version is version('4.3', '>='))) else '' }}"
+        - "{{ '--disable-system-checks' if (pipeline_version == 'nightly' or pipeline_version is version('4.6', '>=')) else '--tuning development' }}"
```

Comment:
This will break on upgrade pipelines :(

See https://github.com/theforeman/forklift/pull/1492 for a similar issue.